data/ui/dialogs/list_edit.ui

```diff
@@ -30,7 +30,7 @@
             </child>
             <child>
               <object class="AdwSwitchRow" id="hide_from_home_row">
-                <property name="title" translatable="yes">Hide these posts from home</property>
+                <property name="title" translatable="yes">Hide Posts From Home</property>
```

Comment:
```suggestion
                <property name="title" translatable="yes">Hide Posts from Home</property>
```

GNOME Calendar uses lowercase "from":

![image](https://github.com/GeopJr/Tuba/assets/50847364/3cc22303-3817-46bf-a435-f04b3e9f2c52)

Reply:
Hey thanks for the review! According to the HIG, every word longer than four letters should be title case.

Reply:
Do you have a link for that?

Reply:
https://developer.gnome.org/hig/guidelines/writing-style.html#header-capitalization

Reply:
Thanks